pyEpiabm/pyEpiabm/tests/test_parameters.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import pyEpiabm as pe
+
+
+class TestParameters(unittest.TestCase):
+    """Test the 'Parameters' class.
+    """
+    def test_instance(self):
+        pe.Parameters.instance()
+
+    def test_setattr(self):
+        pe.Parameters.instance().abc = 1
```

Comment:
what is the actual test? You can maybe collapse the setattr and getattr into one instead

Reply:
@NicholasFan235 can you advise on this - I never really understood this file!

Reply:
These tests mostly just for code coverage completeness.
Parameters is setup as a singleton, i.e. 'Better practice version of a global variable'
Essentially you setup a global variable, but it is managed by a class so you don't have to worry about whether it has been created.
These tests are just checking that you can read and write to the global Parameters object. (mostly for coverage completeness)

Reply:
The `setattr` unit test doesn't affect coverage and is effectively repeated in the `getattr` unit test, so I agree they should probably be combined and have changed that